example/mix.exs

```diff
@@ -19,10 +20,21 @@ defmodule Example.MixProject do
     ]
   end
 
+  def releases do
+    [
+      example: [
+        overwrite: true,
+        quiet: true,
+        steps: [&Shoehorn.Release.init/1, :assemble]
+      ]
+    ]
+  end
+
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:shoehorn, path: "../"}
+      {:shoehorn, path: "../"},
+      {:distillery, "~> 2.1", github: "mobileoverlord/distillery", branch: "release-mutable", override: true},
```

Comment:
Does the example app require using your fork and branch or can distillery be used directly like in the other mix files here?

Reply:
ah, missed this one, thanks